db/schema.rb

```diff
@@ -13,8 +13,6 @@
 ActiveRecord::Schema.define(version: 2019_02_22_063449) do
 
   # These are extensions that must be enabled in order to support this database
-  enable_extension "pg_buffercache"
```

Comment:
Is this change expected?

Reply:
it's not related to this work, so if we want to do it we should do that as a separate PR if we want it so we can easily roll it back

Reply:
Right. This shouldn't be here. I'm sill not entirely clear on why it showed up. Good spot.

Reply:
discussed IRL,
this isn't a migration so won't affect prod
happy to include this to eliminate the diff